bedrock/mozorg/templates/mozorg/about/policy/transparency/jul-dec-2016.html

```diff
@@ -0,0 +1,256 @@
+            </th>
+            <th scope="col">
+              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-counter-notice' }}">{{ _('Counter Notices') }}</a>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">{{ _('Firefox Add-ons') }}</th>
+            <td>4</td>
+            <td>0</td>
+          </tr>
+          <tr>
+            <th scope="row">{{ _('Firefox Marketplace (Apps)') }}</th>
+            <td>0</td>
+            <td>0</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>{{ _('Trademark') }}</h3>
+      <p>{{ _('In the Reporting Period, we received 65 Trademark Takedown Notices and 1 Counter Notice') }}</p>
+
+      <table class="table">
+        <thead>
+          <tr>
+            <th scope="col">{{ _('Mozilla Service') }}</th>
+            <th scope="col">
+              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-takedown-notice' }}">{{ _('Takedown Notices') }}</a>
+            </th>
+            <th scope="col">
+              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-counter-notice' }}">{{ _('Counter Notices') }}</a>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">{{ _('Firefox Add-ons') }}</th>
+            <td>65</td>
+            <td>1</td>
+          </tr>
+          <tr>
+            <th scope="row">{{ _('Firefox Marketplace (Apps)') }}</th>
+            <td>0</td>
+            <td>0</td>
+          </tr>
+        </tbody>
+      </table>
+
+    </div>
+  </section>
+
+  <section class="section" id="supplement">
+    <h2 class="section-title" data-accordion-role="tab">{{ _('Supplement') }}</h2>
+
+    <div class="content" data-accordion-role="tabpanel">
+      <h3>{{ _('Legislative Reform') }}</h3>
+
+      <p>
+      {% trans
+          shared_responsibility='https://blog.mozilla.org/blog/2016/09/13/cybersecurity-is-a-shared-responsibility/',
+          security_vuln='https://blog.mozilla.org/netpolicy/2016/09/19/improving-government-disclosure-of-security-vulnerabilities/',
+          speaker_gov='http://cyberlaw.stanford.edu/events/government-hacking-vulnerabilities-equities-process',
+          speaker_hack='http://cyberlaw.stanford.edu/events/government-hacking-rule-41',
+          gag_orders='https://blog.mozilla.org/mozilla-brief-of-joint-amicus-in-msft-v-doj/',
+          eu_privacy='https://blog.mozilla.org/netpolicy/files/2017/01/Mozilla-Filing_E-Privacy-Survey.pdf',
+          india_cloud='https://blog.mozilla.org/netpolicy/files/2016/07/Mozilla-Submissions-TRAI-Consultation-on-Cloud-Computing.pdf'
+      %}
+        In the Reporting Period, we highlighted the need to see
+        <a href="{{ shared_responsibility }}" rel="external">cybersecurity as a shared responsibility</a>,
```

Comment:
This is a tiny nit and I doubt these particular links will likely ever change, but I've recently started to question if we should be including `rel="external"` inside trans blocks in general. The URLs can change independently of translations, however this bakes in the assumption that the URL will only ever be external and provides little real user benefit. I think here it's likely fine, but just flagging in general.